pallets/claims/src/lib.rs

```diff
@@ -475,7 +489,13 @@ pub mod pallet {
 				// The weight of this logic is included in the `claim_attest` dispatchable.
 				// </weight>
 				Call::claim_attest { dest: account, signer, signature, statement } => {
-					let data = account.using_encoded(to_ascii_hex);
+					let data = match account {
```

Comment:
Can we make this into a helper function so we can deduplicate it?

Reply:
Sure!